### DISCUSSION TITLE
8.0.0

### DISCUSSION THREAD

**Discussion by mithunsatheesh** [mithunsatheesh/node-rules#112] [General]
Typescript rewrite and support.
Package is being sanity tested now.

<hr /><em>This discussion was created from the release <a href='https://github.com/mithunsatheesh/node-rules/releases/tag/'>8.0.0</a>.</em>

**Comment by nsingh2510** [comment]
Any update on when the 8.0.0 will be released.

**Reply by nsingh2510** [reply]
@mithunsatheesh  Any update on same ?

**Comment by mithunsatheesh** [comment]
follow up in #75. 

@OmgImAlexis has made the changes available in below branch
https://github.com/mithunsatheesh/node-rules/tree/8.0.0

**Reply by nsingh2510** [reply]
Hi @mithunsatheesh  , @OmgImAlexis  any update when the new 8.0.0 branch will be released?

**Reply by OmgImAlexis** [reply]
Sorry, tomorrow. Was busy with work.

**Reply by nsingh2510** [reply]
@OmgImAlexis  Thanks :) Hope this tomorrow comes soon :D

**Reply by OmgImAlexis** [reply]
Let's get it done today. I honestly can't really focus on anything bigger with all the russia/ukraine stuff happening right now.

**Reply by OmgImAlexis** [reply]
@mithunsatheesh do you have automated publishing setup in github and/or do you usually publish manually? Might be worth setting up a github action today todo that.

**Reply by mithunsatheesh** [reply]
sorry @OmgImAlexis. i used to publish manually. ill fix it.

**Reply by nsingh2510** [reply]
@mithunsatheesh  and @OmgImAlexis  Hope it gets released this week.

**Reply by mithunsatheesh** [reply]
closing since this is complete.
